tests/testthat/test-compiler-pt-to-lines.r

```diff
@@ -0,0 +1,14 @@
+acontext("pt/points to lines conversion test")
+
+viz <- list(
+  p1 = ggplot()+
+    geom_point(aes(Petal.Length, Sepal.Length, color=Species),data=iris))
+
+theme.pars <- plot_theme(viz)
+panel_margin_lines <- pt.to.lines(theme.pars$panel.margin)
+
+if(grid::unitType(theme.pars$panel.margin) == "pt" || grid::unitType(theme.pars$panel.margin) == "points") {
```

Comment:
why do you need this if????
would be easier to understand (less repetitive) using %in% rather than ||, 
```r
if(something %in% c(val1, val2))
```

Reply:
Got you! I'll rectify it!